Commands/Monitor.php

```diff
@@ -212,21 +220,57 @@ private function getIterationsFromArg()
     }
 
     /**
-     * Loads the `perpage` argument from the commands arguments.
+     * Loads the `rowperpage` argument from the commands arguments.
      *
      * @return int|null
      */
     private function getPerPageFromArg()
     {
-        $perPage = $this->getInput()->getOption('perpage');
+        $perPage = $this->getInput()->getOption('rowperpage');
         if (!is_numeric($perPage)) {
-            throw new \Exception('perpage needs to be numeric');
+            throw new \Exception('rowperpage needs to be numeric');
         } else {
             $perPage = (int)$perPage;
             if ($perPage <= 0) {
-                throw new \Exception('perpage needs to be a non-zero positive number');
+                throw new \Exception('rowperpage needs to be a non-zero positive number');
             }
         }
         return $perPage;
     }
+
+    /**
+     * Loads the `jumptopage` argument from the commands arguments.
+     *
+     * @return int|null
+     */
+    private function getJumpToPageFromArg()
+    {
+        $perPage = $this->getInput()->getOption('jumptopage');
+        if (!is_numeric($perPage)) {
+            throw new \Exception('jumptopage needs to be numeric');
+        } else {
+            $perPage = (int)$perPage;
+            if ($perPage <= 0) {
+                throw new \Exception('jumptopage needs to be a non-zero positive number');
+            }
+        }
+        return $perPage;
+    }
+
+    /**
+     *
+     * @return bool
+     */
+    private function interactiveCapability()
+    {
+        if (str_starts_with(strtoupper(PHP_OS), 'WIN')) {
```

Comment:
@haristku Can you simplify this function to
```suggestion
        return function_exists('readline_callback_handler_install');
```